### ISSUE TITLE
Add Database Rule - Enrolled students in canceled courses

### ISSUE THREAD

**Issue by andreabringhurst** [ut-effectiveness/utValidateR#83]
Add a database rule that shows enrolled students in canceled courses - data steward is Julie.

**Comment by andreabr-dixie** [comment]
Here is the banner SQL:
--canceled courses with enrollments
select ssbsect_term_code, ssbsect_crn, ssbsect_subj_code, ssbsect_crse_numb, ssbsect_seq_numb, ssbsect_ssts_code, ssbsect_enrl
from SSBSECT
where ssbsect_term_code >= '202240'
and ssbsect_ssts_code = 'C'
and ssbsect_enrl > '0';

**Comment by jpfeifer-dsu** [comment]
The sql on the validation checks are looking at active courses only.  ssbsect_ssts_code ='A'.  We're going to postpone this for now until we have more information from Vicki on what is causing this error.

**Comment by cdemkegit** [comment]
@andreabringhurst looking at the last message above from Justin,

> The sql on the validation checks are looking at active courses only. ssbsect_ssts_code ='A'. We're going to postpone this for now until we have more information from Vicki on what is causing this error.

Whos do we need to talk to about this if it is still an issue?

**Comment by andreabringhurst** [comment]
@cdemkegit This is still an issue. The tricky part of this, is we can't use Edify tables for this because Edfiy only pulls in active courses (as Justin said above). We have to use banner tables. If this isn't fixed, this would mean that banner Headcount is different from Edify headcount. Do we have any data in the pins that is pulling from Banner tables and not Edify tables? Do we need to create something that pulls in banner tables?

**Comment by CSPierce01** [comment]
@MattNickodemus @cdemkegit @andreabringhurst We could update our sql to use the section and student_section tables.
```
SELECT b.student_id,
       a.course_id,
       a.course_reference_number,
       a.term_id,
       a.subject_code,
       a.course_number,
       a.section_number,
       a.section_status_code,
       a.section_status_desc
FROM quad.section a
FULL JOIN quad.student_section b ON a.course_id = b.course_id
WHERE a.section_status_desc = 'Cancelled'
  AND b.is_enrolled = TRUE;
```
I didn't add a term_id parameter, but we can.

**Comment by andreabringhurst** [comment]
I know why Justin excluded canceled classes from the checks (only selected Active classes). Most of the checks we do have missing data if they are canceled (i.e. instructors, meeting times and locations, etc.). I think we found that some of the checks were giving errors on canceled classes, so we just excluded them. Now there is a check that needs to be done on canceled classes. @cdemkegit when running the check (lets say for missing instructors), can we add R code that only includes Active classes (AND ssbsect_ssts_code = 'A')? Is there some better work around?
